homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -20,11 +24,19 @@
 )
 from .entity import PlugwiseEntity
 
-BINARY_SENSOR_MAP = {
-    "dhw_state": ["Domestic Hot Water State", None],
-    "slave_boiler_state": ["Secondary Heater Device State", None],
-}
 SEVERITIES = ["other", "info", "warning", "error"]
+BINARY_SENSORS: tuple[BinarySensorEntityDescription, ...] = (
+    BinarySensorEntityDescription(
+        key="dhw_state",
+        name="DHW State",
```

Comment:
I previous name was `Domestic Hot Water State` - Looking at the code the name was never used and was generated by the key, leaving this comment here for others but I think this change is valid.

Reply:
Yeah, one would think that, but that values wasn't used actually...
So it matches the actual value it previously had now :)

Reply:
If we already rename the other one, should we rename this to `Domestic Hot Water State`?

Reply:
🤷  I don't think that is really needed. The other rename is not done for the looks 😉